modules/nodes-cluster-overcommit-project-disable.adoc

```diff
@@ -7,29 +7,60 @@
 [id="nodes-cluster-overcommit-project-disable_{context}"]
 = Disabling overcommitment for a project
 
+ifndef::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
 When enabled, overcommitment can be disabled per-project. For example, you can allow infrastructure components to be configured independently of overcommitment.
+endif::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
 
-.Procedure
+ifdef::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
+If required by your use case, you can disable overcommitment on any project that is not managed by Red Hat. For a list of projects that cannot be modified, see "Red Hat Managed resources" in _Support_.
```

Comment:
@laubai I am not seeing "Red Hat Managed resources" in the Support book. Am I looking in the wrong place?

Reply:
It's in Support under Troubleshooting -  https://docs.openshift.com/rosa/support/troubleshooting/sd-managed-resources.html

This should probably read "in _Additional resources_" instead

Reply:
Can you use the full title: _Red Hat OpenShift Service on AWS managed resources_? I went looking for a section called _Red Hat Managed resources_ and couldn't find one. If you want to leave as is, let me know and we can merge directly!

Reply:
I may update the title of that section, honestly - the point is that this list of resources, which are in ROSA, are managed by RH rather than the customer - will update shortly and refresh the PR

Reply:
Updated the title and intro of the section and confirmed that no other references to this section needed to be updated as a result